components/annot-discrete-choices.vue

```diff
@@ -100,17 +109,28 @@
 
         computed: {
 
-            displayTable() {
+            saveButtonEnabled() {
```

Comment:
Thanks! And agreed with docstring and semicolons. What do you mean with 

>  alphabetize functions within template subgroups

Do you mean sorting them alphabetically? If so, is this something we can get the linter to worry about? Otherwise this seems like an easy thing to forget

Reply:
Yes, I mean sorting them alphabetically. Just found the rule that can be used. https://eslint.vuejs.org/rules/sort-keys.html

I believe this was from the Vue style suggestions - and it's something I just practice by habit for the sake of findability.

Reply:
Hhm, I didn't find it in the vue style guide as a recommended rule. Having thought about it a little bit, one downside I see to this is that thematically related functions could only be grouped together if we name them alphabetically. Ideally this isn't a big problem because we keep the number of methods small in the components. If you like to have this be a consistent rule, we should probably make a new issue and then set this both in the linter and document it somewhere for future contributors.

Reply:
Yeah, I just tried this out and I think it's not a good idea. 

[e.g. here](https://github.com/neurobagel/annotation_tool/blob/master/pages/annotation.vue#L136)
```javascript

                // Getters
                columnDescription: this.columnDescription,
                isMissingValue: this.isMissingValue,
                valueDescription: this.valueDescription,
                // Store fields
                columnToCategoryMap: this.columnToCategoryMap,
                dataDictionary: this.dataDictionary,
                dataTable: this.dataTable,
                missingColumnValues: this.missingColumnValues
```

Would raise an error because the thematically grouped declarations violate alphabetical ordering. If you think this is still worth it @jarmoza, I'd say make an issue and work out these cases and then we can discuss it?